chaoslib/litmus/platform/gke/node_cpu_consumption.yml

```diff
@@ -75,7 +75,7 @@
 
     ## GIVING CPU SPIKE
     - name: Giving CPU spike ...
-      shell: kubectl exec -it {{ pod_ds_app }} -n {{ a_ns }} -- bash -c "stress --cpu {{ cpu_limit }} --timeout {{ c_duration }}; exit"
+      shell: kubectl exec -it {{ pod_name }} -n {{ a_ns }} -- bash -c "stress --cpu {{ cpu_limit }} --timeout {{ c_duration }}; exit"
```

Comment:
Can you wrote this command inside the job itself and wait for the completion of the job instead of using exec command. refer container-kill exp